pages/deployment/[uuid].tsx

```diff
@@ -188,11 +190,25 @@ export default function Deployment() {
     <>
       <Wrapper title={`Deployment: ${deployment.metadata.highLevelDetails.name}`} page='deployment'>
         {hasUploadType && initialVersionRequested?.metadata.buildOptions.uploadType === ModelUploadType.Zip && (
-          <Box sx={{ textAlign: 'right', pb: 3 }}>
-            <Button variant='outlined' color='primary' startIcon={<Info />} onClick={handleClickOpen}>
-              Show download commands
-            </Button>
-          </Box>
+          <Grid container spacing={1}>
```

Comment:
I would change this to use a <Stack /> component instead https://mui.com/components/stack/

There's nothing wrong with using <Grid /> (as we've used it in various places in the app), but I'd say that for "simple" in-line layouts, Stack is better. Grid is better when dealing with larger layouts where you're sorting out layouts both vertically _and_ horizontally.